tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -25,6 +25,7 @@
     # name, policy, action noise pct, success rate
     ['assembly-v1', SawyerAssemblyV1Policy(), .0, 1.],
     ['basketball-v1', SawyerBasketballV1Policy(), .0, .98],
+    ['box-close-v1', SawyerBoxCloseV1Policy(), .0, .85],
```

Comment:
any idea why this is limited to 85%? is there a structural error in the environment? does it need corrections and a v2?

Reply:
It's just a difficult environment. The target box (on which the sawyer has to place the lid) is toward the far end of the table. Because the gripper's quaternion is locked in `[-1, 0, 1, 0]`, the sawyer has to do some arm gymnastics to reach that far/that high (it's IK solver struggles a bit). In doing so, the lid sometimes gets dropped. The policy attempts to pick it back up, but it likes to get stuck on corners of the box.

To fix this, the simplest thing would be to limit how far forward the target box can be, but I wasn't sure if 85% success was low enough to warrant changes.

Reply:
i think we should limit it and make that a v2 change. we really want these envs to all be ~95% solvable, or else i makes other analysis harder.

Reply:
i guess you already merged this so -- can you make an issue about it